pkg/plugins/builtin/http/README.md

```diff
@@ -39,6 +39,10 @@ action:
         user: {{.config.basicAuth.user}}
         password: {{.config.basicAuth.password}}
       bearer: {{.config.auth.token}}
+      mTLS:
+        root_ca: {{.config.mtls.rootca}}
```

Comment:
Even tho its pretty common to include a chain of certs in the same document, it may be worh mentioning it. It directly refers to the usage of `AppendCertsFromPEM` later.

Reply:
Regarding `root_ca`, I guess I shouldn't have put in here.
This is not linked to mTLS feature, this only comes to be able to provide another Root Certificate Authority while calling a distant service.
Maybe we can leave this configuration to the end-user, by adding the corresponding Root CA into the system trust-store, and let uTask relies on the system trust store.
What do you think ?

Reply:
I think it can still be handy to specify a CA at runtime with this parameter. Not all usescases relies on the system trust-store.